gadgets/src/transaction/tests.rs

```diff
@@ -83,31 +91,36 @@ fn test_is_valid_transaction_spend() {
 			let mut prover_transcript = Transcript::new(b"Transaction");
 			let mut prover = Prover::new(&pc_gens, &mut prover_transcript);
 
-			let (com_input_inverse_val, var_input_inverse_val) = prover.commit(input_inverse.clone(), Scalar::random(&mut test_rng));
+			let (com_input_inverse_val, var_input_inverse_val) =
```

Comment:
would rather the args break the line so:
```
let (com_input_inverse_val, var_input_inverse_val) = prover.commit(
        input_inverse.clone(),
        Scalar::random(&mut test_rng)
);
```
possible?

Reply:
Looks like it's not. It first breaks the line after the `=` sign and then looks at the arguments